exasol_script_languages_container_ci/lib/release_uploader.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import re
+from tempfile import TemporaryDirectory
+from typing import Tuple
+
+from exasol_script_languages_container_ci.lib.asset_uploader import AssetUploader
+from exasol_script_languages_container_ci.lib.ci_export import CIExport
+
+
+def _parse_repo_url(source_repo_url: str) -> str:
+    """
+    source_repo_url is expected to have the following format: `https://github.com/exasol/script-languages-repo`
+    where `exasol/script-languages-repo` is the repository for which the release will be created.
+    This method returns the repository id.
+    """
+    res = re.search(r"^https://github.com/([a-zA-Z0-9\-_/]+)$", source_repo_url)
+    if res is None:
+        raise ValueError("Parameter source_repo_url is in unexpected format.")
```

Comment:
Describe expected format in the error message

Reply:
done